Formula/b/beancount.rb

```diff
@@ -92,8 +93,8 @@ class Beancount < Formula
   end
 
   resource "pdfminer2" do
-    url "https://files.pythonhosted.org/packages/55/18/0a856200ff54db80612f66878f9b037ddb9c873f885bc58339b528611994/pdfminer2-20151206.tar.gz"
-    sha256 "7d05aa3dd1e779080fef13aef454501b51a3f7649d7f18e78c640bdbd34e1e77"
+    url "https://files.pythonhosted.org/packages/e0/55/5e235321d7494772264b577a8569c102b9d9ef867f7239d14d562e89bed9/pdfminer2-20151206.macosx-10.10-x86_64.tar.gz"
+    sha256 "9c0599bfde105a8d58e3f679c31ab84871dc9bd3debf1be6511e1abaa4db867f"
```

Comment:
idk what's happening here, but this change will need to be reverted

Reply:
`pdfminer2` appears to have 2 source distributions, probably back from when PyPI was much more permissive about such things: https://pypi.org/project/pdfminer2/20151206/#files

So this is possibly a `pip` bug. I can look into it a bit more in a bit.

Reply:
The plot thickens: `pdfminer2` isn't listed as an extra requirement in the formula mapping, and I don't see it listed as a dependency in `beancounter`'s source either: https://github.com/beancount/beancount

(It looks like beancounter shells out to `pdf2txt.py` opportunisitically, but doesn't actually specify pdfminer2 as a dep. That maybe means it could be replaced with https://github.com/pdfminer/pdfminer.six)

Reply:
Never mind, it apparently is a dependency somewhere: https://inspector.pypi.io/project/beancount/2.3.6/packages/24/9f/68d0fae833527b8ac2660070fd2dc1ad26bdcfbf99537032c033442c5cba/beancount-2.3.6-cp310-cp310-macosx_10_9_x86_64.whl/beancount-2.3.6.dist-info/METADATA

Reply:
Mystery solved: it looks like it's probably tests-only, and was removed recently: https://github.com/beancount/beancount/commit/6a5458175e3734ac63a9976a6852997d238346e6

Given that, we can probably safely exclude it via the formula mapping.

Reply:
Yeah, I was kind of manually excluding it from the dependency updates. 😅

Reply:
Maybe it's worth adding an exclusion rule here so you don't have to keep doing that? :slightly_smiling_face: 

https://github.com/Homebrew/homebrew-core/blob/master/pypi_formula_mappings.json